doc/api/errors.md

```diff
@@ -903,6 +903,11 @@ Encoding provided to `TextDecoder()` API was not one of the
 
 `--print` cannot be used with ESM input.
 
+<a id="ERR_EVENT_RECURSION"></a>
+### `ERR_EVENT_RECURSION`
+
+Thrown when an attempt is made to recursively dispatch an event on `EventTarget`.
```

Comment:
Why would this be an error?

Reply:
Because it causes an infinite recursion. In chrome try each of the following:

```js
const et = new EventTarget();
et.addEventListener('foo', (event) => et.dispatchEvent(event));
et.dispatchEvent(new Event('foo'));
// Reports DOMException, event already being dispatched
```

```js
const et = new EventTarget();
et.addEventListener('foo', (event) => et.dispatchEvent(new Event('foo')));
et.dispatchEvent(new Event('foo'));
// Causes infinite recursion
```

Reply:
Firefox also prevents this case but doesn't crash.

Reply:
So the idea is to introduce spec divergences to avoid infinite recursion? Why? I assume Node doesn't prevent people from writing infinitely-recursive functions; why prevent them from writing infinitely-recursive event invocations?